src/components/BlogPost/PostNavigation.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+
+import { FaCaretLeft, FaCaretRight } from 'react-icons/fa';
+
+const PostNavigation = ({ prev, next }) => (
+  <div className="flex justify-center h-24">
+    { prev
+          && (
+            <Link to={prev.frontmatter.path}>
+              <div className="flex-1 text-center px-4 py-2 m-2">
```

Comment:
Maybe give this one a try: For me it aligned the items (caret and text) more nicely.
What do you think?

```suggestion
              <div className="flex text-center items-center px-4 py-2 m-2">
```